lhotse/cut.py

```diff
@@ -34,8 +35,60 @@
 AnyCut = Union['Cut', 'MixedCut', 'PaddingCut']
 
 
+class CutCommonsMixin:
```

Comment:
I find the `Commons` part of the name confusing.  IDK if this is a common pattern you  got from somewhere?
Maybe CutUtilsMixin or something?  And perhaps comment what it's intended for, i.e. what classes are supposed to use it an what general kind of functionality it adds?

Reply:
Eh, it's a mess really. I've just tried to turn it into an abstract base class for all Cut classes, but `dataclasses` module authors did not bother to support ABCs. For now, I've changed the name, moved `mix` and `append` in here (as their implementations are identical in all types of cuts), and added a comment explaining this somewhat weird design choice.

As to `Commons`, only after your comment I've realized it has a well-defined meaning in English and is a bad substitute for "common/shared parts", which is not as terse. `Utils` it is, at least for now.